scripts/assemble-changelog.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# for some reason you can't just put a heredoc in a variable...
+read -r -d '' USAGE << EOF
+assemble-changelog.sh PACKAGE VERSION
+  Assembles the changelog for PACKAGE at VERSION.
+EOF
+
+if [ "$#" == "0" ]; then
+  echo "$USAGE"
+fi
```

Comment:
Oh wait.  Don't you need `exit 1` or something here?  I think that just prints out the message and keeps going.  It's similar in the other scripts.